usage/local-api.md

```diff
@@ -30,8 +30,7 @@ The Local API is one method of obtaining data from YouTube. For another method o
 | ---------------------------------------------------------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------- | -------------------------------- |
 | [youtubei.js](https://github.com/LuanRT/YouTube.js)                                      | MIT     | Search Functionality + Suggestions, Playlists, Trending, Obtain Video Information, Generate DASH Files, Live Chat | No                               |
 | [videojs-vtt-thumbnails-freetube](https://github.com/FreeTubeApp/videojs-vtt-thumbnails) | MIT     | Handle Video Thumbnails / Storyboards                                                                             | Yes                              |
-| [yt-channel-info](https://github.com/FreeTubeApp/yt-channel-info)                        | ISC     | Channel Info / Search                                                                                             | Yes                              |
-| [yt-comment-scraper](https://github.com/FreeTubeApp/yt-comment-scraper)                  | GPL-3.0 | Comment Info / Sort                                                                                               | Yes                              |
+| [yt-channel-info](https://github.com/FreeTubeApp/yt-channel-info)                        | ISC     | Channel Info / Search                                                                                             | Yes                              |                                                                                               | Yes                              |
```

Comment:
Oof
![image](https://user-images.githubusercontent.com/1018543/214201771-a810824c-7c71-4715-a7d9-9e5339267095.png)

Reply:
Big oof indeed. Is it possible for u to commit to this pr to fix it? I already turned off my machine and i am in snorlax mode

Reply:
Fine
![image](https://user-images.githubusercontent.com/1018543/214207209-54876ab6-6374-439b-a808-2bb9286ed758.png)